src/index.ts

```diff
@@ -246,15 +246,16 @@ export class Toolkit {
       'GITHUB_EVENT_PATH',
       'GITHUB_WORKSPACE',
       'GITHUB_SHA',
-      'GITHUB_REF'
+      'GITHUB_REF',
+      'GITHUB_TOKEN'
```

Comment:
does that mean it will now log a warning if `GITHUB_TOKEN` is not set? Is it set to an empty string if I don’t add it to the secrets settings in my workflow file?

Reply:
Yes - but that's not correct, I'll remove it from the list.